travis/main.sh

```diff
@@ -40,6 +38,14 @@ check_git_status() {
 build_autotools() {
 	v --fold="bootstrap" ./bootstrap.sh
 
+	case ${TRAVIS_BUILD_TYPE} in
+	sanitizers)
+		SANITIZE_FLAGS="-fsanitize=address -fsanitize=leak -fsanitize=alignment -fsanitize=bool -fsanitize=enum -fsanitize=float-cast-overflow -fsanitize=float-divide-by-zero -fsanitize=function -fsanitize=integer-divide-by-zero -fsanitize=null -fsanitize=object-size -fsanitize=return -fsanitize=signed-integer-overflow -fsanitize=unreachable -fsanitize=unsigned-integer-overflow"
```

Comment:
Good enough for me :)
On Jun 17, 2016 11:06 AM, "Mike Frysinger" <notifications@github.com> wrote:

> In travis/main.sh
> <https://github.com/libgd/libgd/pull/235#discussion_r67458430>:
>
> > @@ -40,6 +38,14 @@ check_git_status() {
> >  build_autotools() {
> >  	v --fold="bootstrap" ./bootstrap.sh
> >
> > +	case ${TRAVIS_BUILD_TYPE} in
> > +	sanitizers)
> > +		SANITIZE_FLAGS="-fsanitize=address -fsanitize=leak -fsanitize=alignment -fsanitize=bool -fsanitize=enum -fsanitize=float-cast-overflow -fsanitize=float-divide-by-zero -fsanitize=function -fsanitize=integer-divide-by-zero -fsanitize=null -fsanitize=object-size -fsanitize=return -fsanitize=signed-integer-overflow -fsanitize=unreachable -fsanitize=unsigned-integer-overflow"
>
> i think we should add a dedicated option so people can do it independently
> of anything else. in general i feel like people think of doing
> things/combos that i don't ;).
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/libgd/libgd/pull/235/files/090b985e1f89a373e0a8e67f81feb79efd2a681a#r67458430>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AARPKEtd3a4bZKbFF3l0mekbfch31c-fks5qMh0kgaJpZM4I3ZqB>
> .
>